--- configure
+++ configure
@@ -420,7 +420,7 @@
 
 sed "s/\\\$MAJOR/$MAJVER/" $SRCDIR/liblsmash.v > liblsmash.ver
 # Add non-public symbols which have lsmash_* prefix to local.
-find $SRCDIR/common/ $SRCDIR/importer/ -name "*.h" | xargs sed -e 's/^[ ]*//g' | \
+find $SRCDIR/common/ $SRCDIR/importer/ -name '*.h' | xargs sed -e 's/^[ ]*//g' | \
     grep "^\(void\|lsmash_bits_t\|uint64_t\|int\|int64_t\|lsmash_bs_t\|uint8_t\|uint16_t\|uint32_t\|lsmash_entry_list_t\|lsmash_entry_t\|lsmash_multiple_buffers_t\|double\|float\|FILE\) \+\*\{0,1\}lsmash_" | \
     sed -e "s/.*\(lsmash_.*\)(.*/\1/g" -e "s/.*\(lsmash_.*\)/\1;/g" | xargs -I% sed -i "/^};$/i \           %" liblsmash.ver
 # Get rid of non-public symbols for the cli tools from local.
@@ -479,16 +479,17 @@
 SLIB_CMD = $SLIB_CMD
 EOF
 
-cat config.mak
-
-cat >> config.mak << EOF
+cp -f config.mak config.mak.head
+cat > config.mak.part2 << EOF
 SRCS = $SRCS
 SRC_TOOLS = $SRC_TOOLS
 TOOLS_ALL = $TOOLS_ALL
 TOOLS = $TOOLS_NAME
 MAJVER = $MAJVER
 EOF
-
+cat config.mak.head config.mak.part2 > config.mak.merged
+mv -f config.mak.merged config.mak
+rm -f config.mak.head config.mak.part2
 
 for tool in $TOOLS; do
     cat >> config.mak2 << EOF
